docs/agents/ifm_sbn246_flowmeter.rst

```diff
@@ -0,0 +1,92 @@
+Monitoring is performed by connecting the flowmeter device to an IO-Link master
+device from the same company; the querying of flowmeter data is done via
+HTTP requests to the IO-Link master.
+
+.. argparse::
+    :filename: ../socs/agents/ifm_sbn246_flowmeter/agent.py
+    :func: add_agent_args
+    :prog: python3 agent.py
+
+Configuration File Examples
+---------------------------
+
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+OCS Site Config
+```````````````
+
+To configure the SBN Flowmeter Agent we need to add a FlowmeterAgent
+block to our ocs configuration file. Here is an example configuration block
+using all of the available arguments::
+
+      {'agent-class': 'FlowmeterAgent',
+       'instance-id': 'flow',
+       'arguments': [['--ip-address', '10.10.10.159'],
+                     ['--daq-port', '2']]},
+
+.. note::
+    The ``--ip-address`` argument should use the IP address of the DAQ device you've now
+    configured to be on the network.
+
+Docker Compose
+``````````````
+
+The SBN246 Flowmeter Agent should be configured to run in a Docker container. An
+example docker-compose service configuration is shown here::
+
+  ocs-flow:
+    image: simonsobs/socs:latest
+    hostname: ocs-docker
+    network_mode: "host"
+    volumes:
+      - ${OCS_CONFIG_DIR}:/config:ro
+    environment:
+      - INSTANCE_ID=flow
+      - SITE_HUB=ws://127.0.0.1:8001/ws
+      - SITE_HTTP=http://127.0.0.1:8001/call
+      - LOGLEVEL=info
+
+The ``LOGLEVEL`` environment variable can be used to set the log level for
+debugging. The default level is "info".
+
+Description
+-----------
+
+The SBN246 Flowmeter is a device from IFM Electronic, and will be used to
+monitor the flow and temperature of the cooling loops at the site. This
+monitoring is critical, as any change in flow or temperature will allow SO
+to immediately diagnose/anticipate DR behavior. The Agent communicates to
+the flowmeter via an AL1340 IFM Electronic device--an IO-Link master from which the
+agent makes HTTP requests. The flowmeter plugs into 1 of 4 ports on the IO-Link
+master, and the agent queries data directly from that IO-Link master port.
+This is only possible when an ethernet connection is established via the IO-Link
+master's IoT port.
+
+IO-Link Master Network Setup
+````````````````````````````````
+After plugging into the IoT port on your IO-Link master, the IP address of the IO-Link
+device can be configured using the LR Device software. Once the LR Device software
+is downloaded, click the 'read from device' on the upper right (leftmost IOLINK
+button). Once the software has found your IO-Link device, click on the device
+name (AL1340) on the column on the left side (not the ports that may say P1,
+P2... or the name of the device connected to any of the 4 ports on your IO-Link
+master). You can now change  the IP address by clicking the Network (DHCP) button.
```

Comment:
Still a few things here:

- The IP address is for the IO-Link master, not the individual devices.
- "click on the device name" is tricky because of the keywords "master" and "device" here, especially since we really want the "master" in this case. Also, do we need to even find the devices in order to change the IP address of the master?
- Extra space in the last sentence.

Reply:
Yeah do tell me if you get to the software today before I do, but I think when you hover your mouse over the button that finds the IO-Link master, it says "read from device," which if true, is confusing for this documentation, I agree. 

Definitely the software doesn't need to find the devices plugged into the master; think it just does that anyway when it finds the IO-Link master. But my inclination was that we need to click that "read from device" button so that the software can actually find the IO-Link master and tell us its ip.